05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+#| label: apistrat_des
+des_apistrat <- apistrat_slim %>%
+  as_survey_design(strata = stype, weights = pw, fpc = fpc)
+
+des_apistrat
+summary(des_apistrat)
+```
+
+When printing the object, it is specified as a "Stratified Independent Sampling design" also known as a stratified SRS and the strata variable is included. In the summary, a numeric summary of the probabilities of selection are displayed as well as the sample and population stratum sizes.
+
+### Clustered sampling
+
+- Description: In clustered sampling, a population is divided into mutually exclusive subgroups called clusters or primary sampling units (PSUs). These PSUs are randomly sampled and then sampling is done within these clusters. There can be multiple levels of this selection. Clustered sampling is often used when a list of the entire population is not available or data collection involves interviewers needing direct contact with respondents.
+- Requirements: Must have a way to divide the population into clusters. These are commonly structural such as institutions (e.g., schools, prisons) or geographic (e.g. states, counties). 
+- Advantages: Clustered sampling is advantageous when data collection is done in person so interviewers are sent to specific sampled areas rather than completely at random in a country. With cluster sampling, a list of the entire population is not necessary. For example, if sampling students, you do not need a list of all students but only a list of all schools. Once the schools are sampled, lists of students can be obtained within the sampled schools.
+- Disadvantages: Compared to a simple random sample, for the same sample size, clustered samples generally have larger variance estimates.
+- Examples: 
+  - Example 1: Consider a study needing a sample of 6th grade students in the United States, no list likely exists of all these students. Obtaining a list of schools that have 6th graders is more likely to be possible so a study design could select a random sample of schools that have 6th graders. The selected schools can then provide a list of students to do a second stage of sampling where 6th grade students are randomly sampled within each of the sampled schools. This is a one-stage sample design and will be the type of design we will discuss in formulas below.
+  - Example 2: A more complicated but realistic example is necessary when sending out interviewers to households for a survey. Counties could be selected as the PSU, then Census block groups within counties selected as the secondary sampling unit (SSU), and then households within the block groups. This type of design reduces the travel necessary for interviewers.
+
+#### The math {-} 
+
+Consider a population where the are $N$ clusters and $n$ clusters are sampled via SRSWOR. Units within the sampled cluster are sampled via SRSWOR as well. Let $M_i$ be the number of units in cluster $i$ and $\bar{y}_i$ be the sample mean of cluster $i$. Then, a ratio estimator of the population mean is:
```

Comment:
You don't use SRSWOR above, you just call it SRS.  Recommend using that here, or updating the above to specify WOR.